documentation

```diff
@@ -1,51 +0,0 @@
-Requirements:
-
-database - name, age, number of people in household, address, post code
-reatlational database needed for this
-table of postcode ----- which have addressess ----- each addrss has the number of people livign there and the owner --- the number of poeple is connected with the names and age of the poeple livign there
-
-
-
-
-
-
-
-functianility:
-search people- need a get
-search houses- get request
-specific people and house - get request
-
-
-
-
-
-Resoponse Codes:
-code 1: Information that the server has seen your reqeuest
-code 2: Success, 
-code 3: Invalid parameters/redirect
-code 4: Client Error
-code 5: Server Error
-
-
-
-
-Getting started use this url
-URL: https://RCapi.net - has some information about the website and how it works
-
-
-
-/house/		Get     action- show 
-should recieve a code 2, so 200 ok
-
-for invalid entries, recieve a code 400 bad request
```

Comment:
Perhaps examples of invalid entries, what exactly is an invalid entry with regards to this API?